.travis.yml

```diff
@@ -2,22 +2,20 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.2.5
-  - 2.3.1
+  - 2.4.9
+  - 2.5.7
+  - 2.6.5
```

Comment:
There is no ruby specific code, so it should be good enough to just test the minimum supported and latest release.  That is generally enough to make sure the library uses features that are available for all the supported versions.

The oldest ruby version isn't going to be compatible with the newest rails release, so we might just want to use a few of [named jobs](https://docs.travis-ci.com/user/customizing-the-build#naming-jobs-within-matrices) to cover minimum supported ruby & rails versions, latest released ruby and rails version and maybe a final one to cover rails master.